package.json

```diff
@@ -13,20 +13,20 @@
   "bin": {
     "dts-generator": "bin/dts-generator"
   },
+  "scripts": {
+    "prepublishOnly": "tsc --module umd --target es2015 --moduleResolution node index.ts bin/dts-generator.ts",
+    "test": "tests/run.sh"
+  },
   "dependencies": {
-    "bluebird": "3.3.3",
-    "glob": "7.0.0",
-    "mkdirp": "0.5.1"
+    "glob": "^7.1.3",
+    "mkdirp": "^0.5.1"
   },
   "devDependencies": {
-    "dojo-loader": ">=2.0.0-beta.5",
-    "intern": "~3.0.0",
-    "tsd": "0.6.5",
-    "tslint": "3.11.0",
-    "typescript": "^2.1.4"
-  },
-  "scripts": {
-    "prepublish": "tsc --module umd --target es5 --moduleResolution node typings/tsd.d.ts index.ts bin/dts-generator.ts",
-    "test": "tests/run.sh"
+    "@types/glob": "^7.1.1",
+    "@types/mkdirp": "^0.5.2",
+    "dojo-loader": "^2.0.0-beta.7",
```

Comment:
Do we still need the loader? I’m guessing we never released a non-beta version?

Reply:
Build and tests run fine without `dojo-loader`; I removed it now.